bin/tachyon-workers.sh

```diff
@@ -15,7 +15,7 @@ if [ $# -le 0 ]; then
 fi
 
 bin=`cd "$( dirname "$0" )"; pwd`
-DEFAULT_LIBEXEC_DIR="$bin"/../libexec
```

Comment:
but bin/tachyon-workers.sh is never executed because $bin/tachyon-workers.sh is not resolved properly